src/System.Management.Automation/security/wldpNativeMethods.cs

```diff
@@ -494,9 +494,8 @@ private static SystemEnforcementMode GetDebugLockdownPolicy(string path)
             // Support fall-back debug hook for path exclusions on non-WOA platforms
             if (path != null)
             {
-                // Assume everything under SYSTEM32 is trusted, with a purposefully sloppy
-                // check so that we can actually put it in the filename during testing.
-                if (path.Contains("System32", StringComparison.OrdinalIgnoreCase))
```

Comment:
This check is weak, but it appears to only be used for test cases and not exploitable beyond that